### ISSUE TITLE
when going up in the elevator it kills you

### ISSUE THREAD

**Issue by Owned67** [WheteThunger/BetterElevators#2]
Hello,
when going up in the elevator it kills you

if you unload BetterElevator, problem disapear..

Thanks for a fix please

**Comment by WheteThunger** [comment]
Thanks for reporting. I will check it out later today.

Can you share with me what settings you are using, such and speed? Can you also try different settings to narrow down under which circumstances it happens?

In the future, I suggest reporting issues on the uMod forums so that others have visibility into the issue and can chime in.

**Comment by Owned67** [comment]
```json
{
  "DefaultMaxFloors": 6,
  "MaxFloorsRequiringPermission": [
    10,
    15,
    20,
    40
  ],
  "RequirePermissionForPowerless": true,
  "RequirePermissionForLiftCounter": true,
  "MaintainLiftPositionWhenHeightChanges": false,
  "EnsureConsistentOwner": true,
  "EnableSpeedOptions": true,
  "DefaultSpeed": {
    "BaseSpeed": 1.5,
    "EaseType": "Linear"
  },
  "SpeedsRequiringPermission": [
    {
      "Name": "2x",
      "BaseSpeed": 3.0,
      "EaseType": "Linear"
    },
    {
      "Name": "4x",
      "BaseSpeed": 6.0,
      "EaseType": "Linear"
    },
    {
      "Name": "1x.quadratic",
      "BaseSpeed": 0.86,
      "EaseType": "Quadratic"
    },
    {
      "Name": "1.5x.quadratic",
      "BaseSpeed": 1.29,
      "EaseType": "Quadratic"
    },
    {
      "Name": "2x.quadratic",
      "BaseSpeed": 1.72,
      "EaseType": "Quadratic"
    },
    {
      "Name": "1x.cubic",
      "BaseSpeed": 0.72,
      "EaseType": "Cubic"
    },
    {
      "Name": "2x.cubic",
      "BaseSpeed": 1.44,
      "EaseType": "Cubic"
    }
  ],
  "StaticElevators": {
    "EnableCustomSpeed": false,
    "Speed": {
      "BaseSpeed": 3.5,
      "EaseType": "Linear"
    },
    "EnableLiftCounter": false
  }
}
```

Im config manager, can't make better, owner have removed plugin now :(

Ive asked him:
Author: Can you also try different settings to narrow down under which circumstances it happens?
☣Psychoڲнιηι☆۩08/10/2021, 22:09:10
it is still happening i think it's a facePunch thing
ianhendry8008/10/2021, 22:09:50
It’s not plug-in because I removed so it’s a facepunch issue

**Comment by WheteThunger** [comment]
Here's a temporary plugin to address the issue.
https://github.com/WheteThunger/ElevatorFallFix

**Comment by Owned67** [comment]
Thanks, ive sent to owner ;)

**Comment by WheteThunger** [comment]
This issue has been mostly resolved in v1.2.2.
